insights/client/apps/ansible/playbook_verifier/__init__.py

```diff
@@ -1,37 +1,139 @@
 import os
+import copy
+import yaml
+import base64
+import requests
+import gnupg
+import tempfile
+from logging import getLogger
+from requests.exceptions import HTTPError
+from insights.client.utilities import get_version_info
 
-__all__ = ("verify", "PlaybookValidationError")
+__all__ = ("verify", "PlaybookVerificationError")
 
+SIGKEY = 'insights_signature'
+PUBLIC_KEY_FOLDER = './insights/client/apps/ansible/playbook_verifier/public.gpg'    # Update this when we have the key generated
```

Comment:
You may want to use `pkgutil` to load static files here, I believe this won't work once it's a zipped egg. See `insights/client/map_components.py` for an example.

Reply:
@gravitypriest Now using pkgutil to load static files.